lib/backend/routes/github.py

```diff
@@ -837,19 +841,28 @@ def timeline():
     for repo in repos:
         milestones.extend(repo.get('milestones', []))
 
+    def getCirclesId(milestone):
+        milestones_circles = db_session.query(Milestone_circle).filter(
+            Milestone_circle.milestone_id == milestone['id']).all()
+        return [{'circle_id': assoc.circle_id} for assoc in milestones_circles]
+
     def milestoneDateToIso(milestone):
         if milestone.get('due_on'):
             milestone['due_on'] = milestone['due_on'].isoformat()
+        milestone['is_in_edition'] = False
+        milestone['circles'] = getCirclesId(milestone)
         return milestone
 
+    results = [milestoneDateToIso(milestone) for milestone in milestones
```

Comment:
peutetre plus lisible comme ca


`def super_if(milestone):`

    return (
        (milestone.get('due_on')
         and date_from_iso(start) <= milestone['due_on'] < date_from_iso(stop)) # noqa
        or (without_due_date
            and not milestone.get('due_on') # noqa
            and not milestone.get('closed_at'))) # noqa


    results = [milestoneDateToIso(milestone) for milestone in milestones
               if super_if(milestone)]

Reply:
oui c'est mieux, merci :)